docs/contributing-to-keyshade/running-things-locally/docker-support.md

```diff
@@ -0,0 +1,93 @@
+---
+description: Instructions on how to build & run each part of keyshade using docker
+---
+
+# Docker Support
+
+This document provides instructions on how to build and run our applications using Docker.
+
+## Prerequisites
+
+- Docker must be installed on your system.
+
+## Available Applications
+
+We have Docker support for the following applications:
+
+1. Web Application
+2. API
+3. Platform
+
+## Building and Running Applications
+
+### Web Application
+
+To build the web application:
+
+```bash
+pnpm docker:build:web
+```
+
+To run the web application:
+
+```bash
+pnpm docker:run:web
+```
+
+### API
+
+To build the API:
+
+```bash
+pnpm docker:build:api
+```
+
+To run the API:
+
+```bash
+pnpm docker:run:api
+```
+
+### Platform
+
+To build the platform:
+
+```bash
+pnpm docker:build:platform
+```
+
+To run the platform:
+
+```bash
+pnpm docker:run:platform
+```
+
+## Port Usage
+
+The applications use the following ports by default:
+
+- Web Application: 3000
```

Comment:
I think the ports are wrong. @kriptonian1 can you please drop the correct ports?

Reply:
I went around the codebase when I wrote this to find these values..
Just rechecked and they seem OK.

Waiting on input for this

Reply:
I looked this up.

Sadly, both the web and platform packages use the same port number when running the built version. We would need to change that somehow. Would you be able to come up with different mappings in the Dockerfile? Ideally, we would like to map Web to 6969 and Platform to 3025.

Reply:
Am I missing something here?

This is the web docker:
![image](https://github.com/user-attachments/assets/8f47e506-9919-4fef-8dae-a67bbe124d97)

and this is the platform docker:
![image](https://github.com/user-attachments/assets/01755fbc-f660-455f-a9a7-52bc362a085c)

They both expose different ports, but I may be missing something here

Reply:
Hmm, thats strange. No worries, I will get this merged otherwise the scope might be tampered